bpfd-api/src/bpfd.v1.rs

```diff
@@ -101,6 +112,15 @@ impl ProgramType {
             ProgramType::Tracepoint => "TRACEPOINT",
         }
     }
+    /// Creates an enum from field names used in the ProtoBuf definition.
```

Comment:
Hrm now we almost don't need this function :) https://github.com/redhat-et/bpfd/blob/main/bpfd/src/command.rs#L64

Anyways I'll leave for now but I think wherever we can use new autogenerated helpers the better